### DISCUSSION TITLE
How to add user-facing documentation to CronWorkflows

### DISCUSSION THREAD

**Discussion by janeisklar** [argoproj/argo-workflows#13254] [Q&A] [answered]
I did not find a possibility to add a documentation to a cron workflow. Is this not supported?
It would be nice to add a little description to explain what that job does to make it easier for someone looking through the list of cronjobs.

How do you guys solve this? I guess I could add an annotation, but it wouldn't show up in the UI and won't be nicely formatted so it probably is not ideal for this.

**Comment by agilgur5** [comment, marked as answer]
There is a feature in 3.6 (which is unreleased) that has certain annotations show up in the UI: https://github.com/argoproj/argo-workflows/issues/12644. You can try it out by using the `:latest` Server image.

**Reply by janeisklar** [reply]
Thanks a lot, I somehow missed that when searching the issues.

For completeness, these are the annotations in question:
https://github.com/argoproj/argo-workflows/blob/7c805fefeebf35b84f2f4927fd9f8ba4e885350f/ui/src/app/shared/annotations.ts#L2-L3

**Reply by agilgur5** [reply]
It's also documented on this page: https://argo-workflows.readthedocs.io/en/latest/title-and-description/, which will include `CronWorkflows` once the respective PRs are merged.
